release-doc-in-process/all_vf.TTL

```diff
@@ -323,19 +323,57 @@ vf:containedIn a            owl:ObjectProperty ;
         rdfs:comment        "Used when a stock economic resource contains units also defined as economic resources." .
 
 vf:hasBeginning a           owl:DatatypeProperty ;
-        rdfs:domain         time:TemporalEntity ;
+        rdfs:domain         [ owl:unionOf (
+                                time:TemporalEntity
+                                vf:Commitment
+                                vf:EconomicEvent
+                                vf:Intent
+                                vf:Proposal
+                                vf:Scenario
+                                vf:Process) ] ;
         rdfs:range          xsd:dateTimeStamp ;
         vs:term_status      "unstable" ;
         owl:propertyChainAxiom (time:hasBeginning time:inXSDDateTimeStamp) ;
         rdfs:comment        "The planned or actual beginning of a flow or process." .
 
 vf:hasEnd a                 owl:DatatypeProperty ;
-        rdfs:domain         time:TemporalEntity ;
+        rdfs:domain         [ owl:unionOf (
+                                time:TemporalEntity
+                                vf:Commitment
+                                vf:EconomicEvent
+                                vf:Intent
+                                vf:Proposal
+                                vf:Scenario
+                                vf:Process) ] ;
         rdfs:range          xsd:dateTimeStamp ;
         vs:term_status      "unstable" ;
         owl:propertyChainAxiom (time:hasEnd time:inXSDDateTimeStamp) ;
         rdfs:comment        "The planned or actual end of a flow or process." .
 
+vf:hasPointInTime a         owl:DatatypeProperty ;
+        rdfs:domain         [ owl:unionOf (
+                                time:Instant
+                                vf:Commitment
+                                vf:EconomicEvent
+                                vf:Intent) ] ;
+        rdfs:range          xsd:dateTimeStamp ;
+        vs:term_status      "unstable" ;
+        owl:equivalentProperty  time:inXSDDateTimeStamp ;
+        rdfs:comment        "The planned or actual time of a flow; can be used instead of hasBeginning and hasEnd, if so, hasBeginning and hasEnd should be able to be returned with this value." .
+
+time:hasDuration
+        rdfs:domain         [ owl:unionOf (vf:RecipeProcess vf:ScenarioDefinition) ] .
+
+vf:due a                    owl:DatatypeProperty ;
```

Comment:
I thought we had "vf:before" instead of "vf:due", as we also have "vf:after"?

Reply:
I added vf:due as a totally separate thing from owl time stuff.  I don't actually know of much of a use case for vf:after so thought best to wait and see if there is one.  But maybe due isn't the best name anyhow, it is somewhat more specific.

Reply:
OK, I see. "vf:due" seems to have a more clear semantic. The we should either keep vf:before and vf:after and see how the user community will use it, or not.

Reply:
Yes, I agree on seeing how the user community thinks about it.  I plan on working as closely as possible with the current groups implementing VF so I can get feedback as close to the source as possible.  (Small note: We never had vf:before and vf:after, we just used time:before and time:after.)  I'm not clear which you prefer, so let's talk about this next time we chat.

Reply:
Decided to keep vf:due for now, pending more user involvement in current projects.